ansible/roles/stack-sunbird/templates/ml-survey-service.env

```diff
@@ -53,7 +53,7 @@ USER_PROFILE_FETCH_ENDPOINT={{ml_survey_user_profile_fetch_endpoint | default("/
 #   AZURE_ACCOUNT_KEY=xx123supersecretpassword
 #   AZURE_STORAGE_CONTAINER=slassessmentscontainer
 
-{{ml_survey_cloud_config}}
+{{ml_cloud_config}}
```

Comment:
What is this? How will we define this in private? where is the default for this? is the a dictionary, list, string, integer etc?